config/swagger.yaml

```diff
@@ -3156,9 +3156,6 @@ definitions:
       tx:
         description: Unsigned transaction object
         $ref: '#/definitions/EncodedHash'
-      tx_hash:
```

Comment:
Following removal of this item, shall (encoded) hash of (posted) signed tx be returned by `PostTx` request? (So that user can easily call `GetTx`.)

Reply:
I was considering this, but I think it makes sense for the user of the API to construct it themselves. It is easy, and the hash should probably not be trusted from the node anyway.